### PR TITLE
added ctrl+c functionality, terminate procedures.

### PR DESCRIPTION
**Pull request by namrata013**
there was a merge conflict coz of a 3-way merge, resolved it.

### CODE REVIEW

**Comment by abhigyank** [comment]
Hey, good work on this. One issue is - 
run login command. Enter email press enter. Then while entering password press ctrl+c. That has some weird behaviour with the password not getting cancelled. Can you check that?

File: public/js/script.js [hunk 1]
```diff
@@ -161,27 +184,44 @@ $('textarea').keyup(function(e) {
...
+  else if(e.which==67 && ctrl)
+  {
+    if(email==1)
+    {
+      $('#email').remove();
+      $('#root').show();
+      email=0;
+      login=0;
+    }
+      $('.terminal-output').append('<div class="command" role="presentation" aria-hidden="true"><div style="width: 100%;"><span class="user">root@' + username + ': ~$ </span><span>' + command + '</span></div></div>');
```
Comment (abhigyank) [inline public/js/script.js]:
```suggestion
      $('.terminal-output').append('<div class="command" role="presentation" aria-hidden="true"><div style="width: 100%;"><span class="user">root@' + username + ': ~$ </span><span>' + command +  '^C' + '</span></div></div>');
```

Comment (namrata013) [reply public/js/script.js]: ok checking

**Comment by abhigyank** [comment]
Good work @namrata013 . Merged. Thanks! :rocket:

**Pull request merged**